plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSink.java

```diff
@@ -496,6 +496,7 @@ private FileWriter createParquetFileWriter(Path path)
                     identityMapping,
                     compressionCodecName,
                     trinoVersion,
+                    Optional.empty(),
```

Comment:
Why do we skip Delta Lake validation?

Reply:
Because the new parquet writer is already enabled by default there. 
There needs to be a discussion about it's need there and if it is necessary there, then it should be implemented in a follow up. I don't intend to do it in this PR.